doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -762,9 +762,16 @@ Modem libraries
 
 * :ref:`nrf_modem_lib_readme`:
 
-  * Updated the RTT trace backend to allocate the RTT channel at boot, instead of when the modem is activated.
-  * Rename the nRF91 socket offload layer from ``nrf91_sockets`` to ``nrf9x_sockets`` to reflect that the offload layer is not exclusive to the nRF91 Series SiPs.
   * Removed support for deprecated RAI socket options ``SO_RAI_LAST``, ``SO_RAI_NO_DATA``, ``SO_RAI_ONE_RESP``, ``SO_RAI_ONGOING``, and ``SO_RAI_WAIT_MORE``.
+  * Added the `nrf91-modem-trace-rtt` snippet to enable modem tracing using the RTT trace backend.
```

Comment:
```suggestion
  * Added the :ref:`nrf91-modem-trace-rtt <nrf91_modem_trace_rtt_snippet>` snippet to enable modem tracing using the RTT trace backend.
```
Added link to the snippet docs.
We could move this entry to "Developing with nRF91 Series" as we mention about the snippet there. https://docs.nordicsemi.com/bundle/ncs-latest/page/nrf/releases_and_maturity/releases/release-notes-changelog.html#developing_with_nrf91_series.